erpnext/erpnext_integrations/taxjar_integration.py

```diff
@@ -112,7 +112,8 @@ def get_tax_data(doc):
 		'to_street': to_address.address_line1,
 		'to_state': to_shipping_state,
 		'shipping': shipping,
-		'amount': doc.net_total
+		'amount': doc.net_total,
+		'plugin': '[bloomstack]'
```

Comment:
Should this be configurable at user end or hard coded like it is now?

Reply:
its just a parameter to check which software is used for taxjar API. there is no need to configure this.
https://developers.taxjar.com/integrations/certification-checklist/